pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -2044,7 +2044,7 @@ var _ = Describe("Manager", func() {
 		// we need the non-typecast object to make the function we want to test available
 		libvirtmanager := manager.(*LibvirtDomainManager)
 
-		_, err := libvirtmanager.GetGuestInfo()
+		_ := libvirtmanager.GetGuestInfo()
 		Expect(err).ToNot(HaveOccurred())
```

Comment:
This expectation can be removed! Should we check something else?

Reply:
I missed this, thanks.
Fixed.